### PR TITLE
Fixes #29475 - migrate foreman_scap_client_facts.rb script to python

### PR DESCRIPTION
**Pull request by snagoor**

### CODE REVIEW

File: files/foreman_scap_client_facts.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+
+import json
+from optparse import OptionParser
```
Comment (xprazak2) [inline files/foreman_scap_client_facts.py:4]: Is `optparse` needed? I do not see it being used anywhere...

**Comment by xprazak2** [comment]
Works nicely, I just left one comment.

File: files/foreman_scap_client_facts.py (hunk 1 above)
Comment (snagoor) [reply files/foreman_scap_client_facts.py]: @xprazak2 ack updated the code.

**Comment by snagoor** [comment]
@xprazak2 Fixed all issues based on `pylint` and it is happy now. Can you please take a look at it again?

~~~
$ pylint files/foreman_scap_client_facts.py 

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

~~~

**Comment by snagoor** [comment]
@xprazak2 can you please take a look at it.

**Comment by xprazak2** [comment]
Apologies it takes me so long, it looks good. I'll just give it one more spin to be sure and then I'll merge.

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**